### ISSUE TITLE
[Feature Request] Support for ready-to-use GPT3.5

### ISSUE THREAD

**Issue by chaojian-zhang** [betalgo/openai#116]
**Irrelevant Remarks**

Great library by the way, and as someone who does not like to deal with REST APIs, having it exposed as native methods makes my code cleaner.

**Is your feature request related to a problem? Please describe.**

According to [official API doc](https://platform.openai.com/docs/models/overview) and [Engadget news](https://www.engadget.com/openai-will-let-developers-build-chatgpt-into-their-apps-204737530.html), ChatGPT's model is officially available as of today (2023.03.01). It would be nice if this library can support the corresponding names for the completion API.

**Describe the solution you'd like**

I think there should be minimal API change - essentially adding an enumeration. This feature request only concerns the completion API.

**Describe alternatives you've considered**

I tried to make a fork but the `ModelNameBuilder` programming model is a bit confusing to me so I will leave this feature implementation to the original author😂

**Additional context**

![image](https://user-images.githubusercontent.com/7077098/222296336-e2302f1d-e6fd-4c73-9c62-da3013ac0cc5.png)

**Comment by Frogley** [comment]
It's [on the way](https://github.com/betalgo/openai/discussions/113)

**Comment by kayhantolga** [comment]
Sorry about ModelNameBuilder, it's a bit messy but It is really hard to follow OpenAI naming rules :D however, check version 6.7.0
